app/src/main/java/org/openimis/imispolicies/domain/PolicyRenewal.java

```diff
@@ -0,0 +1,178 @@
+package org.openimis.imispolicies.domain;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import java.util.Date;
+
+public class PolicyRenewal implements Parcelable {
+
+    private final int id;
+    @NonNull
+    private final String uuid;
+    private final int policyId;
+    private final int officerId;
+    @NonNull
+    private final String officerCode;
+    @NonNull
+    private final String chfId;
+    @NonNull
+    private final String lastName;
+    @NonNull
+    private final String otherNames;
+    @NonNull
+    private final String productCode;
+    @Nullable
+    private final String productName;
+    @Nullable
+    private final String villageName;
+    @NonNull
+    private final Date renewalPromptDate;
+    @Nullable
+    private final String phone;
+
+    public PolicyRenewal(
+            int id,
+            @NonNull String uuid,
+            int policyId,
+            int officerId,
+            @NonNull String officerCode,
+            @NonNull String chfId,
+            @NonNull String lastName,
+            @NonNull String otherNames,
+            @NonNull String productCode,
+            @Nullable String productName,
+            @Nullable String villageName,
+            @NonNull Date renewalPromptDate,
+            @Nullable String phone
+    ) {
+        this.id = id;
+        this.uuid = uuid;
+        this.policyId = policyId;
+        this.officerId = officerId;
+        this.officerCode = officerCode;
+        this.chfId = chfId;
+        this.lastName = lastName;
+        this.otherNames = otherNames;
+        this.productCode = productCode;
+        this.productName = productName;
+        this.villageName = villageName;
+        this.renewalPromptDate = renewalPromptDate;
+        this.phone = phone;
+    }
+
+    protected PolicyRenewal(Parcel in) {
+        id = in.readInt();
+        uuid = in.readString();
```

Comment:
if you agree with UUID type:
```suggestion
        uuid = java.util.UUID.fromString(in.readString());
```